build.gradle

```diff
@@ -96,4 +120,4 @@ tasks.withType(Copy) {
     includeEmptyDirs = true
 }
 
-defaultTasks 'clean', 'build', 'systemtest'
+defaultTasks 'clean', 'build', 'systemtest', 'uat', 'cypress'
```

Comment:
Perhaps can remove `cypress` from the defaultTasks, since `uat` will already run the frontend tests, and the cypress window needs to be manually run and close for the gradle task to complete, which can be abit troublesome.

Reply:
`cypress` is useful for writing new tests or debugging them. 🤔

Reply:
>`cypress` is useful for writing new tests or debugging them.

Yes, the `cypress` command can be useful for those tasks, but for our `defaultTasks` I prefer if we keep it as a convenient way to test the current code implementation, not really for debugging.

Reply:
Turns out my understanding of `defaultTasks` was incorrect. I thought it was needed to expose `cypress` command. 😊